### PR TITLE
fix script error for SVGElement.setAttributeNS (close #514)

### LINKED ISSUE #514: Regression: script error inside SVGElement.setAttributeNS
see https://testcafe-hhhm.devexpress.com/history?timestamp=1459537468694&siteUrl=http://badoo.com/
Related to https://github.com/DevExpress/testcafe-hammerhead/pull/503

### PR DESCRIPTION
**Pull request by miherlosev**
@churkin @LavrovArtem

### CODE REVIEW

File: src/processing/dom/html-namespaces.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// -------------------------------------------------------------
+
+export default {
+    HTML:   'http://www.w3.org/1999/xhtml',
+    MATHML: 'http://www.w3.org/1998/Math/MathML',
+    SVG:    'http://www.w3.org/2000/svg',
+    XLINK:  'http://www.w3.org/1999/xlink',
+    XML:    'http://www.w3.org/XML/1998/namespace',
+    XMLNS:  'http://www.w3.org/2000/xmlns/'
+};
```
Comment (churkin) [inline src/processing/dom/html-namespaces.js]: I propose to use lowercase

Comment (miherlosev) [inline src/processing/dom/html-namespaces.js]: It named as in https://www.w3.org/TR/2011/WD-html5-20110405/namespaces.html.
So, in parse5 it named simular (https://github.com/inikulin/parse5/blob/24cb16c4a46f84fba01c6f41fc96817a013a6495/lib/common/html.js).

Comment (churkin) [inline src/processing/dom/html-namespaces.js]: We use lowercase format for a constants. @inikulin

Comment (miherlosev) [inline src/processing/dom/html-namespaces.js]: ping @inikulin

Comment (inikulin) [inline src/processing/dom/html-namespaces.js]: @churkin is right. For constant enumerations we use lowercase. It should be always used as 
`HTML_NAMESPACE.html` then. But, I have no idea why we need ALL namespace constants - only two of them used in code. I don't see why we need dedicated file for them. It's just fine to have inline constants in each file in that case.

Comment (miherlosev) [inline src/processing/dom/html-namespaces.js]: > But, I have no idea why we need ALL namespace constants - only two of them used in code.

For future use.
For example, for https://github.com/DevExpress/testcafe-hammerhead/issues/477  issue we will need the XML namespace.

> I don't see why we need dedicated file for them. It's just fine to have inline constants in each file in that case.

The same logical costants (namespaces) grouped in the separate file.

Comment (inikulin) [inline src/processing/dom/html-namespaces.js]: #477 is related to XML

> For future use.

Never add dead code based on assumption. Let's add namespace when it they will be required.

Comment (miherlosev) [inline src/processing/dom/html-namespaces.js]: Ok

Comment (miherlosev) [inline src/processing/dom/html-namespaces.js]: What about it?

> I don't see why we need dedicated file for them. It's just fine to have inline constants in each file in that case.

The same logical costants (namespace constants) grouped in the separate file.

Comment (inikulin) [inline src/processing/dom/html-namespaces.js]: Until they are not used in different files I don't see necessity to move them to the dedicated file

Comment (miherlosev) [inline src/processing/dom/html-namespaces.js]: Ok

**Comment by miherlosev** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Comment by miherlosev** [comment]
ping @inikulin

**Comment by inikulin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**